lib/init/rc.boot

```diff
@@ -167,6 +171,16 @@ main() {
     command -v udevd >/dev/null &&
         udevadm control --exit
 
+    log "Running rc.d scripts..."; {
+        if [ -d /etc/rc.d ] ; then
```

Comment:
The `if [ -d` can be removed as the loop will end on the first iteration if this directory does not exist. It ends up being one `if` test even when removed.

Reply:
Please check 22c98c I think it should be good now